### DISCUSSION TITLE
How to compile graphemer

### DISCUSSION THREAD

**Discussion by March-Wind** [webpack/webpack#17335] [Q&A]
# Bug report

**What is the current behavior?**
  Webpack was unable to package the socks proxy agent package correctly when packaging node projects, and reported an error: `TypeError: Graphemer.default is not a constructor`
**If the current behavior is a bug, please provide the steps to reproduce.**
1. ` webpack.config.js`：
      ```
      module.exports = {
          entry: './src/index.js',
          target:'node',
          output: {
              path: __dirname + '/dist',
              filename: 'bundle.js',
          },
          mode:'development',
          module: {
      
              rules: [
                  {
                      test: /\.js$/,
                      exclude: /node_modules/,
                      use: [{
                        loader: 'babel-loader',
                        options:{
                        presets: ['@babel/preset-env']
                      }}],
                  },
                ]
          },
      }
      ```
2. `src/index.js`:
      ```
      import Graphemer from "graphemer";
      
      console.log(Graphemer);
      
      
      const a = new Graphemer.default();
      console.log(a)
      ```

**What is the expected behavior?**
Able to run compiled files normally

**Other relevant information:**
webpack version:  5.85.1
Node.js version: 16
Operating System: mac os
Additional tools:

**Comment by alexander-akait** [comment]
Don't use `default()`

**Reply by March-Wind** [reply]
If  do not use `default()` to execute `node ./src/index.js` will report an error :`TypeError: Graphemer is not a constructor`

**Reply by alexander-akait** [reply]
Please check your babel configuration and generated code by babel
